<launch>
  <node name="lidar_node"  pkg="lidar_ros"  type="lidar_node" output="screen" respawn="false" >
    <param name="serial_name"                 type="string" value="/dev/ttyUSB0"/>  
    <param name="serial_baud"                 type="int" value="230400"/>  
    <param name="frame_id"                    type="string" value="laser_frame"/>
    <param name="counterclockwise"            type="bool"   value="true"/>
    <param name="angle_min"                   type="double" value="-180.0" />
    <param name="angle_max"                   type="double" value="180" />
    <param name="range_min"                   type="double" value="0.001" />
    <param name="range_max"                   type="double" value="15" />
    <param name="resolution_fixed"            type="bool"   value="false"/>
    <param name="angle_corp_string"           type="string" value="" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser"
    args="0.19 0.0 0.2 0.06 0.0  0.0 /base_footprint /laser_frame 40" />
</launch>
   